cli/command/system/version.go

```diff
@@ -48,16 +52,22 @@ Server:{{if ne .Platform.Name ""}} {{.Platform.Name}}{{end}}
    {{- end}}
   {{- end}}
  {{- end}}
-{{- end}}{{end}}`
+ {{- end}}{{- end}}
+ {{- if .KubernetesOK}}{{with .Kubernetes}}
+ Orchestrator:
+   Kubernetes:	{{.Kubernetes}}
+   Stack:	{{.StackAPI}}{{- end}}{{end}}`
```

Comment:
Minor formatting nit: the `{{- end}}{{end}}` can go on a new line so that the template reads a bit better.

`-` in a template means "remove all whitespace before this token, so the trailing newline wont exist in the final rendered string.

Reply:
Thanks, I wasn't aware of that, fixed.